### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by brandroidian** [jaredhobbs/home-assistant-hx3#3] [General]
I just had a new hvac system installed yesterday and was forced to migrate to the HX3 thermostats and was disappointed I'd potentially lose all of my climate automations. This integration works perfectly!

**Comment by adambulla** [comment]
Another thank you here! I had to switch from Ecobee to HX3 last year to support the variable speed air conditioning, even though I much preferred the Ecobee. I recently started using HA, and this integration is fantastic! Very much appreciated, Jared.
